java/release_notes.txt

```diff
@@ -1,3 +1,14 @@
+May 03, 2016: libphonenumber-7.3.2
+Metadata changes:
+ - Updated phone metadata for region code(s):
+   AU, BR, ET, IN, KH, ML, NO, SB, TJ, US
+ - Updated short number metadata for region code(s): CA, ML, US
+ - New geocoding data for country calling code(s):
```

Comment:
55 (pt) doesn't seem to be new? 1548 and 1743 aren't country calling codes, so drop them from here.

Change it to 55 (en, pt) below and add 1 (en).